precompiles/unified-accounts/UnifiedAccounts.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity ^0.8.0;
+
+/**
+ * @title UA interface.
+ */
+
+/// Interface to the precompiled contract on Shibuya
```

Comment:
Probably better to drop `Shibuya` from the comment.

And the address too, I guess. It's not tied to the interface.

Reply:
Yeah I'll drop Shibuya.
For address, it's kind of tied to interface as we have same address in all networks and we have this in every precompile interface, also easier for people to just use it right from here.

Reply:
It's not tied to the `.sol` file as an interface.

It's tied to the implementation of this interface in our precompiles - there it's said to be the number mentioned above. But it could just as well change in the implementation without any change being required in the `.sol` file.

It's not a big deal, and you are right when you say it's also mentioned in other files.
I consider it to be wrong - however it is convenient.

No need to change it, it was more of a minor comment.